content/projects/data-science-specific/cross-validation-and-simple-linear-regression/_index.md

```diff
@@ -15,6 +15,9 @@ story_points: 13
 submission_type: repo
 tags:
 - simple-linear-regression
+- probability-statistics
+- plotly
+- seaborn
```

Comment:
Are you sure about seabourn and plotly? I cant see them in this project description. I see no mention of visualisation at all. Where did these tags come from?